Tchap/Modules/Authentication/AuthenticationErrorPresentableMaker.swift

```diff
@@ -89,14 +103,42 @@ final class AuthenticationErrorPresentableMaker {
     
     // MARK: - Private
     
-    private func authenticationServerErrorPresentable(from authenticationServiceError: AuthenticationServiceError) -> ErrorPresentable {
+    private func restClientBuilderErrorPresentable(from restClientBuilderError: RestClientBuilderError) -> ErrorPresentable {
         
         let title: String = Bundle.mxk_localizedString(forKey: "login_error_title")
         let message: String
         
-        switch authenticationServiceError {
+        switch restClientBuilderError {
         case .unauthorizedThirdPartyID:
             message = Bundle.mxk_localizedString(forKey: "login_error_forbidden")
```

Comment:
The correct message here is TchapL10n.authenticationErrorUnauthorizedEmail (FYI this change has been already merged on develop)